web/src/features/map/Map.tsx

```diff
@@ -371,65 +329,13 @@ export default function MapPage({ onMapLoad }: MapPageProps): ReactElement {
       style={{ minWidth: '100vw', height: '100vh' }}
       mapStyle={MAP_STYLE as mapboxgl.Style}
     >
-      <Layer id="ocean" type="background" paint={styles.ocean} />
-
-      <Source
-        id="zones-clickable"
-        promoteId={'zoneId'}
-        type="geojson"
-        data={worldGeometries}
-      >
-        <Layer id="zones-clickable-layer" type="fill" paint={styles.zonesClickable} />
-        <Layer id="zones-hoverable-layer" type="fill" paint={styles.zonesHover} />
-        <Layer id="zones-border" type="line" paint={styles.zonesBorder} />
-      </Source>
-      <Source id="states" type="geojson" data={statesGeometries}>
-        <Layer id="states-border" type="line" paint={styles.statesBorder} minzoom={2.5} />
-        <Layer
-          id="state-labels-name"
-          type="symbol"
-          source="states"
-          layout={{
-            'text-field': ['get', 'stateName'],
-            'symbol-placement': 'point',
-            'text-size': 12,
-            'text-letter-spacing': 0.12,
-            'text-transform': 'uppercase',
-
-            'text-font': ['poppins-semibold'],
-          }}
-          paint={{
-            'text-color': 'white',
-            'text-halo-color': '#111827',
-            'text-halo-width': 0.5,
-            'text-halo-blur': 0.25,
-            'text-opacity': 0.9,
-          }}
-          minzoom={4.5}
-        />
-        <Layer
-          id="state-labels-id"
-          type="symbol"
-          source="states"
-          layout={{
-            'text-field': ['get', 'stateId'],
-            'symbol-placement': 'point',
-            'text-size': 12,
-            'text-letter-spacing': 0.12,
-            'text-transform': 'uppercase',
-            'text-font': ['poppins-semibold'],
-          }}
-          paint={{
-            'text-color': 'white',
-            'text-halo-color': '#111827',
-            'text-halo-width': 0.5,
-            'text-halo-blur': 0.25,
-            'text-opacity': 0.9,
-          }}
-          maxzoom={4.5}
-          minzoom={3}
-        />
-      </Source>
+      <Layer
+        id="ocean"
+        type="background"
+        paint={{ 'background-color': theme.oceanColor }}
+      />
```

Comment:
I would consider moving this out as well even if it don't really make a difference code wise. But just so we put all layers as their own files in map-layers.